src/index.ts

```diff
@@ -194,10 +202,8 @@ export default class NewRelicLambdaLayerPlugin {
       return;
     }
 
-    const { exclude = [] } = this.config;
-    if (_.isArray(exclude) && exclude.indexOf(funcName) !== -1) {
-      this.serverless.cli.log(`Excluded function ${funcName}; skipping`);
-      return;
+    if (this.shouldSkip(funcName)) {
+      return
```

Comment:
We may need a semicolon on this line.  (I have an auto-linting PR that should be merged shortly.)